include/eve/detail/function/simd/riscv/bitmask.hpp

```diff
@@ -0,0 +1,39 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/detail/abi.hpp>
+#include <eve/module/core/regular/if_else.hpp>
+
+#include <bitset>
+
+namespace eve::detail
+{
+//================================================================================================
+// Logical to Bits
+//================================================================================================
+template<typename T, typename N>
+EVE_FORCEINLINE auto
+to_bits(rvv_ const&, logical<wide<T, N>> p) noexcept
+requires rvv_abi<abi_t<T, N>>
+{
+  using uint_type = as_uinteger_t<T>;
+  return eve::if_else(p, static_cast<uint_type>(0), static_cast<uint_type>(0));
```

Comment:
this looks like a bug. Is this code used?

Reply:
Fixed. Seems was broken during one of updates.